js/piwik.js

```diff
@@ -3694,6 +3701,35 @@ if (typeof window.Piwik !== 'object') {
                         && performanceAlias.timing.requestStart && performanceAlias.timing.responseEnd) {
                         request += '&gt_ms=' + (performanceAlias.timing.responseEnd - performanceAlias.timing.requestStart);
                     }
+
+                    if (performanceAvailable && !performanceTracked) {
```

Comment:
Not sure... do you reckon it makes sense to track at least some metrics that are already available when eg `trackPageview` is called? And some other metrics that are only available after the load event track them with the next request once `performanceAvailable  = true`? This way would get bit more data and the metrics that were sent become more accurate. Eg `server time`. 
Of course it means we don't always have all information for every user because there might be no second tracking request, but at least there will be some data and more accurate etc.

Reply:
ok. will change it that with a page view all data that is already available will be sent. If not all were already available the next request will send the performance data (again) and update the values.